### ISSUE TITLE
Base θ-adjoint: OBB logit-logit second jet (∂²gate/∂ℓ²) still softmax-shaped (child of #2330)

### ISSUE THREAD

**Issue by SauersML** [SauersML/gam#2371]
## Base θ-adjoint: OBB logit-logit second jet (∂²gate/∂ℓ²) is still softmax-shaped

Spun out of #2330. After the OBB data-block factor fix and the Patch D `∂ΔC/∂θ` legs land, the exact-A θ-adjoint FD arbiter is **exact on the coordinate channel** (rel <1.1e-4) but retains a **weight-independent ~1.43-abs residual on the "signal" logit slot per row** (≈1.8% at the arbiter fixture where fd≈133; ~59% in low-|fd| regimes where fd≈1.43). The arbiter (`sae_exact_a_theta_adjoint_gap_measure_2330_patchd`) asserts `LOGIT_TOL=3e-2` with a comment pointing here; **tighten it to `COORD_TOL=1e-3` when this lands.**

### Why it's not any of the Patch-D legs (ruled out)
- Not channel-2 (ordered-BB prior `∂ΔC_obb/∂ℓ`): the residual is **weight-independent** (identical at `log_lambda_sparse=−4` and `−2`), whereas channel-2 scales `exp(log_lambda_sparse)`. Verified `data.weight = exp(lls)` is correct and channel-2 is exercised by the weighted arbiter variant.
- Not the softmax entropy majorizer: `active_softmax_majorizer_logit_derivative_entry` returns exactly 0 for OBB (`scale=0` → early `continue`).
- Not residual curvature: at small `residual_scale` those legs are ~0, yet the ~1.43 persists.

### Localization (the proof sketch)
`logdet_theta_adjoint_dense`'s `_ =>` data arm forms `∂B[a,b]/∂θ_w = second(a,w)·first(b) + first(a)·second(b,w)`. The **coordinate** slots are exact, and a coordinate slot's Γ sum exercises `second(logit, coord)` — so the mixed second jet is correct. But `second(logit, logit)` (`∂²gate/∂ℓ²·decoder·φ`) appears **only** in logit-direction slots — exactly where the residual lives. So the OBB logit-logit second jet is the suspect.

Candidate root: in `row_jet_program.rs`, `hand_softmax_column` hardcodes `softmax_gate_derivs` for the Logit×Logit second channel. It is test-oracle-only on the surface (called at ~1889/2140/2183), and production OBB routes through `gate_tower`/`PerAtomLogistic` (which is why the coordinate/first-order channels are correct) — so the exact production path that feeds the OBB logit-logit second jet needs tracing in `construction_row_jet_logdet_channels.rs::row_jets_for_logdet`. The fix is the OBB analogue of the data-factor fix: replace the softmax gate second-derivative with the per-atom logistic one (`∂²ζ_k/∂ℓ_k² = ζ_k(1−ζ_k)(1−2ζ_k)/τ²`, cross-atom exactly zero).

### Acceptance
- Arbiter logit channel closes to the coordinate level; tighten `LOGIT_TOL` → `COORD_TOL`.
- Softmax path bitwise unchanged.
- The channel-2-weighted arbiter variant (`sae_exact_a_theta_adjoint_gap_measure_2330_patchd_weighted`) goes green on the logit slots at high OBB weight.

**Comment by SauersML** [comment]
## Hypothesis (i) confirmed + landed in-lane: one-sided cross-atom OBB gate factor

**Root cause.** The `(Some(atom_w), Coord, Coord)` arm of `logdet_theta_adjoint_dense` added the OBB gate-gradient factor to the Gauss-Newton curvature `B[a,b]` only when `atom_w == atom_a == atom_b`, dropping the one-sided cross-atom term as "exactly zero". That is wrong: `B[a,b] = <J_a, J_b>` with each leg carrying its independent gate `g_k = sigma(l_k/tau)` linearly, so
```
dB[a,b]/dl_w = [1(w==a) + 1(w==b)] * (1 - g_w)/tau * B[a,b]
```
Same-atom-both = `sided=2` (bitwise the prior landed value); one-sided cross-atom = `sided=1`, silently dropped.

**Fix.** Replace the same-atom-only condition with `sided = (atom_w==atom_a) as u32 + (atom_w==atom_b) as u32; sided as f64 * (1 - a_soft[atom_w]) * patchd_obb_inv_tau`.

**FD arbiter (`sae_exact_a_theta_adjoint_gap_measure_2330_patchd`), before -> after:**

| channel | before | after |
|---|---|---|
| `max_coord_rel` | 1.11e-4 | **1.11e-4** (unchanged) |
| `max_logit_rel` | ~1.8e-2 | **1.05e-3** |

17x collapse on the logit channel, coord channel bit-untouched, no overshoot -> the derivation is confirmed. The residual ~1e-3 on the asserted fixture (rs=0, prior inert at lls=-6) is FD-floor-limited (central-diff roundoff on the ~4.2e2 signal integrand; abs_err grows as h shrinks 1e-4 -> 1e-5).

**Not fully closed by (i).** The report-only `_weighted` variant (rs=0.005, lls=-4, which activates the residual-curvature DeltaC + OBB prior channels) is UNCHANGED by this fix:
```
PATCHD_W rs=0.005 lls=-4.00 row=0 Logit{atom:0} fd=-1.431910e0 analytic=-2.913e-3 rel=5.876e-1
```
That path goes through `patchd_residual_third_leg` / the exact-A DeltaC block, not the B-block factor edited here. The jet tower (PerAtomLogistic Hessian) and its fill are correct, so this is a distinct **channel-2 (residual-curvature / OBB prior) logit-derivative** defect, not the second-jet fill of the original title. Tracking as remaining work.

**Comment by SauersML** [comment]
Resolved by **88fac3950** (one-sided cross-atom OBB gate factor in the dense θ-adjoint data block).

The titled hypothesis — "OBB logit-logit second jet ∂²gate/∂ℓ² still softmax-shaped" — was **falsified** during investigation: production OBB routes through `gate_tower`/`PerAtomLogistic`, which auto-differentiates σ(ℓ/τ)'s second jet correctly (the softmax-shaped `hand_softmax_column` is test-oracle-only). The real dense-path defect was the Gauss-Newton curvature gate-gradient: `B[a,b]=⟨J_a,J_b⟩` carries each atom's independent gate `g_k` linearly, so `∂B/∂ℓ_w = [1(w==a)+1(w==b)]·(1−g_w)/τ·B` — the one-sided cross-atom term (factor 1) was dropped as "exactly zero."

**Measured result** (FD arbiter `sae_exact_a_theta_adjoint_gap_measure_2330_patchd`): max_logit_rel **1.8e-2 → 1.05e-3** (17× collapse, no overshoot); coord slots bit-unchanged. The remaining 1.05e-3 is central-difference FD floor (abs error grows as h shrinks), not an analytic gap on this fixture. LOGIT_TOL tightened accordingly.

Remaining items, tracked elsewhere (not this issue's titled concern):
- A distinct weighted-fixture (rs>0, lls>−6) residual in the residual-curvature / OBB-prior-channel logit derivatives — needs re-measurement now that the E-attributability unit (e972387215) has landed on the exact-A path.
- BIC-d_eff FD-discreteness (#2398) makes channel-wise FD verification structurally invalid at floor-adjacent fixtures.

Closing: the titled second-jet concern is resolved (falsified + the true dominant defect fixed and measured).
